### PR TITLE
api/topics submitted for review

### PR DESCRIPTION
**Pull request by Joelymodevs**
pretty sure everything is fine, bit confused by wording today but I just think the bank holiday stole my brain away from me

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                    {
+                      description: 'Not dogs',
+                      slug: 'cats'
+                    },
+                    {
+                      description: 'what books are made of',
+                      slug: 'paper'
+                    }
+                  ]
+            )
```
Comment (robert112233) [inline __tests__/endpoints.test.js]: Good work, while this works, there is a better way to be testing arrays: 

This test is too brittle, If any properties are added to each object, you will have to make lots of changes to the test. It is better practice to make the _same_ broad assertion _for each_ (hint) element of the array. That way if in future you add a column in the database you only have to change one assertion. Don't forget to include a length expectation as well!

File: app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+const { displayTopics } = require('./controllers')
+
+
+app.get('/api/topics', displayTopics)
```
Comment (robert112233) [inline app.js]: You might want to consider renaming this, display has connotations of rendering/frontend. I would use something like fetchTopics 👍

File: models.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const db = require('./db/connection');
+
+exports.getTopics = () => {
+    return db.query('SELECT * FROM topics').then((results) => {
+        const display = results.rows.map(row => ({
```
Comment (robert112233) [inline models.js]: I'm not sure why you are mapping through here. You should be able to return result.rows. In your test suite, you'd want to expect each individual property is coming back, not just slug and description 😄

File: __tests__/endpoints.test.js (hunk 1 above)
Comment (robert112233) [reply __tests__/endpoints.test.js]: You should also add a generic 404 invalid endpoint test, you might find express handles this automatically.

**Review by robert112233 — CHANGES_REQUESTED** [review]
Good work Joel, go back and refactor the test to use one assertion for each element in the array. You should also include generic 404 invalid endpoint test.

File: db/seeds/run-seed.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-const devData = require('../data/development-data/index.js');
+const devData = require('../data/test-data/index.js');
```
Comment (JVal-NC) [inline db/seeds/run-seed.js:1]: This should stay as devData, otherwise you'll be changing this file back and fourth between test and dev data

**Pull request merged**